junit-jupiter-api/src/main/java/org/junit/jupiter/api/extension/ConditionEvaluationResult.java

```diff
@@ -45,6 +45,21 @@ public static ConditionEvaluationResult disabled(String reason) {
 		return new ConditionEvaluationResult(false, reason);
 	}
 
+	/**
+	 * Factory for creating <em>disabled</em> results with custom reasons
+	 * added by the user.
+	 *
+	 * @param reason the default reason why the container or test should be disabled
+	 * @param customReason the custom reason why the container or test should be disabled
+	 * @return a disabled {@code ConditionEvaluationResult} with the given reasons
+	 */
+	public static ConditionEvaluationResult disabled(String reason, String customReason) {
+		if (customReason.isEmpty()) {
```

Comment:
:o: Should we use `StringUtils.isBlank()` here?

Reply:
Good idea 👍